apps/chat/src/utils/app/clean.ts

```diff
@@ -78,9 +79,12 @@ export const cleanConversation = (
       conversation.id ||
       constructPath(
         conversation.folderId || getConversationRootId(),
-        conversation.name || DEFAULT_CONVERSATION_NAME,
+        (conversation.name && prepareEntityName(conversation.name)) ||
```

Comment:
we shouldn't use `prepareEntityName` into this method - unexpected update name can happen when we saved other. Use it directly in your high-level method

Reply:
Moved it one level up. Thanks for the advise!